docs/end-user-topics/cgc-launch-with.rst

```diff
@@ -0,0 +1,55 @@
+Cancer Genomics Cloud Launch-With
+=================================
+
+Dockstore integrates with the `Cancer Genomics Cloud <https://cgc.sbgenomics.com>` (CGC)
```

Comment:
Link looks weird in preview at https://beta.docs.dockstore.org/en/feature-2823-seven-bridges/end-user-topics/cgc-launch-with.html

Reply:
Fixed